xsensemulator/emulator.go

```diff
@@ -42,29 +43,29 @@ func NewUDPSerialPort(origin, destination string) (*UDPSerialPort, error) {
 	}, nil
 }
 
-func (t *UDPSerialPort) Read(p []byte) (int, error) {
+func (t UDPSerialPort) Read(p []byte) (int, error) {
 	n, _, err := t.OriginConn.ReadFromUDP(p)
```

Comment:
I suggest either adding an arg for r/w timeout in the constructor OR adding methods for setting them on the emulator (like `emulator.SetDefaultReadTimeout` or something) that you can check here and set on the conn, something like:
```
if t.defaultTimeout != 0 {
  t.OriginConn.SetReadDeadLine(t.defaultTimeout)
}
```

Reply:
removed the methods and added on a constructor level! Very nice to use now